src/backstop/backstop_client.ts

```diff
@@ -183,7 +184,7 @@ export class BackstopClient {
     );
   }
 
-  async updateEmissionCycle(
+  async gulpEmissions(
```

Comment:
Should the sdk also contain gulp pool emissions?

Reply:
No, this can't be called by anyone but the pool